jersey/src/main/java/com/netflix/conductor/server/resources/WorkflowResource.java

```diff
@@ -189,8 +189,8 @@ public void restart(@PathParam("workflowId") String workflowId, @QueryParam("use
     @Path("/{workflowId}/retry")
     @ApiOperation("Retries the last failed task")
     @Consumes(MediaType.WILDCARD)
-    public void retry(@PathParam("workflowId") String workflowId) {
-        workflowService.retryWorkflow(workflowId);
+    public void retry(@PathParam("workflowId") String workflowId,@QueryParam("retryFromLastFailedTask") boolean retryFromLastFailedTask) {
```

Comment:
Naming the parameter as `retryFromLastFailedTask` doesn't seem intuitive. For example, if the workflow doesn't have any subworkflow(s), the default behavior is to retry from last failed task. Setting this parameter `retryFromLastFailedTask` to `false` on retry request would then have no impact.

Instead, how about calling this `resumeSubworkflowTasks` or something similar?

Also, please add `@DefaultValue("false")` to QueryParam (See `restart` definition above for reference).